spec/visual/samples/augeas

```diff
@@ -0,0 +1,32 @@
+(*
+Multiline
+Comment
+*)
```

Comment:
A large amount of the visual sample looked to me to be duplicative. I reduced it to a tighter collection of expressions that test various elements of the lexer.